exercises/robot-simulator/robot_simulator_test.exs

```diff
@@ -0,0 +1,62 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("robot_simulator.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule RobotSimulatorTest do
+  use ExUnit.Case
+
+  test "create has sensible defaults" do
+    assert RobotSimulator.create == { :ok, %RobotSimulator{ direction: :north, position: {0, 0} } }
+  end
+
+  @tag :pending
+  test "create works with valid arguments" do
+    assert RobotSimulator.create(:north, {0, 0}) == { :ok, %RobotSimulator{ direction: :north, position: {0, 0} } }
+    assert RobotSimulator.create(:south, {-10, 0}) == { :ok, %RobotSimulator{ direction: :south, position: {-10, 0} } }
+    assert RobotSimulator.create(:east, {0, 10}) == { :ok, %RobotSimulator{ direction: :east, position: {0, 10} } }
+    assert RobotSimulator.create(:west, {100, -100}) == { :ok, %RobotSimulator{ direction: :west, position: {100, -100} } }
+  end
+
+  @tag :pending
+  test "create errors if invalid direction given" do
+    assert RobotSimulator.create(:invalid, {0, 0}) == { :error, "invalid direction" }
+    assert RobotSimulator.create(0, {-10, 0}) == { :error, "invalid direction" }
+    assert RobotSimulator.create("east", {0, 10}) == { :error, "invalid direction" }
+  end
+
+  @tag :pending
+  test "create errors if invalid position given" do
+    assert RobotSimulator.create(:north, "invalid") == { :error, "invalid position" }
+    assert RobotSimulator.create(:north, 0 ) == { :error, "invalid position" }
+    assert RobotSimulator.create(:north, [0, 0]) == { :error, "invalid position" }
+    assert RobotSimulator.create(:north, nil) == { :error, "invalid position" }
+  end
+
+  @tag :pending
+  test "simulate single robot" do
+    { :ok, robot } = RobotSimulator.create(:east, { -2, 1 })
+
+    assert RobotSimulator.simulate(robot, "RLAALAL") == { :ok, %RobotSimulator{ direction: :west, position: { 0, 2 }}}
+  end
+
+  @tag :pending
+  test "simulate many robots" do
+    { :ok, robot1 } = RobotSimulator.create(:north, { 0, 0 })
+    { :ok, robot2 } = RobotSimulator.create(:east, { 2, -7 })
+    { :ok, robot3 } = RobotSimulator.create(:south, { 8, 4 })
+
+    assert RobotSimulator.simulate(robot1, "LAAARALA") == { :ok, %RobotSimulator{ direction: :west, position: { -4, 1 }}}
+    assert RobotSimulator.simulate(robot2, "RRAAAAALA") == { :ok, %RobotSimulator{ direction: :south, position: { -3, -8 }}}
+    assert RobotSimulator.simulate(robot3, "LAAARRRALLLL") == { :ok, %RobotSimulator{ direction: :north, position: { 11, 5 }}}
```

Comment:
can this test file have a helper function to expand
`ok_sim(:north, {11,5})`
to
`{ :ok, %RobotSimulator{ direction: :north, position: { 11, 5 }}}`
?